packages/python/src/alumnium/server/agents/planner_agent.py

```diff
@@ -20,15 +23,31 @@ class Plan(BaseModel):
 
 
 class PlannerAgent(BaseAgent):
+    NAVIGATE_TO_URL_EXAMPLE = """
+Example 2:
```

Comment:
why 2? Maybe we can skip a number to avoid sudden duplicates in the future?
```suggestion
Example:
```